velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -395,6 +395,14 @@ void HiveDataSink::close() {
   }
 }
 
+int64_t HiveDataSink::getCompletedBytes() const {
+  int64_t completedBytes{0};
+  for (const auto& ioStats : ioStats_) {
+    completedBytes += ioStats->rawBytesWritten();
```

Comment:
Can you also leverage this to update "onDiskDataSizeInBytes" in fragment output with tests? Thanks!